app/views/foreman_bootdisk/generic_host.erb

```diff
@@ -9,6 +9,8 @@
 #
 # Copy this template to customize it, the original is read-only.
 
+echo Foreman Bootdisk: Generic image
```

Comment:
Adding some extra info because I always get screenshot and I don't know which bootdisk causes the error.